en/guide/index.md

```diff
@@ -22,57 +22,58 @@ As a framework, Nuxt.js comes with a lot of features to help you in your develop
 
 ## How it Works
 
-![Vue with Webpack and Babel](https://i.imgur.com/avEUftE.png)
+![Vue with webpack and Babel](https://i.imgur.com/avEUftE.png)
 
 Nuxt.js includes the following to create a rich web application development:
 
-- [Vue 2](https://github.com/vuejs/vue)
-- [Vue-Router](https://github.com/vuejs/vue-router)
-- [Vuex](https://github.com/vuejs/vuex) (included only when using the [store option](/guide/vuex-store))
-- [Vue-Meta](https://github.com/declandewet/vue-meta)
+- [Vue 2](https://vuejs.org/)
+- [Vue Router](https://router.vuejs.org/en/)
+- [Vuex](https://vuex.vuejs.org/en/) (included only when using the [store option](/guide/vuex-store))
+- [Vue Server Renderer](https://ssr.vuejs.org/en/)
+- [vue-meta](https://github.com/declandewet/vue-meta)
```

Comment:
It's seems the product name is the same as package name so Vue-Meta or Vue Meta doesn't exist.